sdk/python/feast/feature_store.py

```diff
@@ -1766,6 +1780,11 @@ def _get_feature_views_to_use(
 
         return views_to_use
 
+    def stop_go_server(self):
+        """Force the Go feature server to shut down."""
+        if enable_go_feature_server(self.config) and self._go_server is not None:
+            self._go_server.stop()
```

Comment:
We should have an `else` here I think. Otherwise what would happen if the user ran this command. Maybe we should raise an exception?

Reply:
Well I can imagine that a user will only call this method if they think they are running a Go server. So I would probably raise an exception if the Go server is not enabled at all, and maybe leave the rest as is.